lib/moab/verification_result.rb

```diff
@@ -15,34 +16,30 @@ class VerificationResult
     attr_accessor :subentities
 
     # @param entity [Object] The name of the entity being verified
-    def initialize(entity)
+    # @param verified [Boolean]
+    # @param details [Hash]
+    def initialize(entity, verified = false, details = nil)
       @entity = entity
-      @verified = false
-      @details = nil
+      @verified = verified
+      @details = details
       @subentities = Array.new
     end
 
-    # @param entity [Symbol] The name of the entity being verified
+    # @param entity [#to_s] The name of the entity being verified
     # @param expected [Object] The expected value
     # @param found [Object] The found value
     # @return [VerificationResult] The result of comparing the expected and found values
     def self.verify_value(entity, expected, found)
-      result = VerificationResult.new(entity.to_s)
-      result.verified = (expected == found)
-      result.details = { 'expected' => expected, 'found' => found }
-      result
+      new(entity.to_s, (expected == found), 'expected' => expected, 'found' => found)
```

Comment:
I don't find this clearer ... too many expressions inside parens.  And, in this case, i would find the curly braces easier than not including them.  Maybe
```
new(entity.to_s, expected == found, { 'expected' => expected, 'found' => found})
```

it's clearer there are three args, anyway.

Reply:
This is a common syntax for anything that accepts a hash as the last param.  Could make the details hash a separate line and pass that to remove confusion though.